rootfs/standard/usr/bin/mynode_reinstall_app.sh

```diff
@@ -88,6 +88,11 @@ elif [ "$APP" = "mempool" ]; then
     rm -rf data
     rm -rf mysql/data
     cd ~
+elif [ "$APP" = "albyhub" ]; then
+    systemctl stop albyhub
+	docker rmi albyhub
+	docker rmi ghcr.io/getalby/hub:$ALBYHUB_VERSION
+    rm -rf /mnt/hdd/mynode/albyhub/.albyhub-data
```

Comment:
Is there a reason to delete the data during a reinstall? I would normally add a button to the settings page to clean an apps data. Uninstall / reinstall normally don't clear out storage on the data drive.